docs/intro/install.rst

```diff
@@ -1,245 +1,56 @@
 .. _intro-install:
 
-==================
-Installation guide
-==================
+============
+Installation
+============
 
-Installing Scrapy
-=================
+Scrapy supports Python 3.5 or above under CPython (the default Python
+implementation) and PyPy (starting with PyPy 5.9).
 
-Scrapy runs on Python 3.5 or above
-under CPython (default Python implementation) and PyPy (starting with PyPy 5.9).
-
-If you're using `Anaconda`_ or `Miniconda`_, you can install the package from
-the `conda-forge`_ channel, which has up-to-date packages for Linux, Windows
-and OS X.
-
-To install Scrapy using ``conda``, run::
+.. _install-pypi:
+.. _intro-install-platform-notes:
+.. _intro-install-ubuntu:
+.. _intro-install-macos:
 
-  conda install -c conda-forge scrapy
+Installing from the Python Package Index
+========================================
 
-Alternatively, if you’re already familiar with installation of Python packages,
-you can install Scrapy and its dependencies from PyPI with::
+You can install Scrapy from the `Python Package Index`_::
 
     pip install Scrapy
 
-Note that sometimes this may require solving compilation issues for some Scrapy
-dependencies depending on your operating system, so be sure to check the
-:ref:`intro-install-platform-notes`.
-
-We strongly recommend that you install Scrapy in :ref:`a dedicated virtualenv <intro-using-virtualenv>`,
```

Comment:
I think that'd be nice to preserve it, as it is not said like this in https://packaging.python.org/tutorials/installing-packages/.